airbyte-webapp/src/pages/connections/CreateConnectionPage/ExistingEntityForm.tsx

```diff
@@ -94,7 +77,7 @@ const ExistingEntityForm: React.FC<IProps> = ({ type, onSubmit }) => {
           }}
         >
           {({ isSubmitting, setFieldValue }) => (
-            <FormContent>
+            <Form className={styles.form}>
```

Comment:
Removed styled components and switched the styling to flex boxes instead of margins, so make adding the BuilderPrompt here easier